src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function add(num) {
+    if (num === undefined) {
```

Comment:
Instead of checking for `undefined`, you could check if `num` is not provided. This would make the code more readable and concise.